src/components/forms/attendee-form/attendee-form.js

```diff
@@ -269,11 +286,12 @@ class AttendeeForm extends React.Component {
                     {entity.id !== 0 && currentSummit.attendee_extra_questions && currentSummit.attendee_extra_questions.length > 0 &&
                     <Panel show={showSection === 'extra_questions'} title={T.translate("edit_attendee.extra_questions")}
                            handleClick={this.toggleSection.bind(this, 'extra_questions')}>
-                        <QuestionAnswersInput
-                            id="extra_questions"
-                            answers={entity.extra_questions}
-                            questions={currentSummit.attendee_extra_questions}
-                            onChange={this.handleChange}
+                        <ExtraQuestionsForm 
+                            extraQuestions={currentSummit.attendee_extra_questions}
```

Comment:
this is not gonna work
bc the collection of extra questions of the summit is not formatted how we need it
u need to call to the extra questions endpoint on on componentdidmount to get the questons